google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/DmlBatch.java

```diff
@@ -119,6 +120,17 @@ public Timestamp getCommitTimestampOrNull() {
     return null;
   }
 
+  @Override
+  public CommitResponse getCommitResponse() {
```

Comment:
Doubles my suspicion that this doesn't belong in the superclass.

Reply:
See above.